src/modules/i18n/containers/I18nProvider.js

```diff
@@ -86,22 +113,38 @@ export class I18nProvider extends React.Component {
   }
 
   render () {
-    const {lateef, openSans, raleway} = this.state.fonts
-    return <I18nextProvider i18n={this.i18n}>
-      <div style={{'direction': RTL_LANGUAGES.includes(this.state.language) ? 'rtl' : 'ltr'}}>
-        <ReactHelmet>
-          {lateef && <link href='/fonts/lateef/lateef.css' rel='stylesheet' />}
-          {openSans && <link href='/fonts/open-sans/open-sans.css' rel='stylesheet' />}
-          {raleway && <link href='/fonts/raleway/raleway.css' rel='stylesheet' />}
-        </ReactHelmet>
-        {this.props.children}
-      </div>
-    </I18nextProvider>
+    const { lateef, openSans, raleway } = this.state.fonts
+    return (
+      <I18nextProvider i18n={this.i18n}>
+        <div
+          style={{
+            direction: RTL_LANGUAGES.includes(this.state.language)
+              ? 'rtl'
+              : 'ltr'
+          }}
+        >
+          <ReactHelmet>
+            {lateef && (
+              <link href='/fonts/lateef/lateef.css' rel='stylesheet' />
+            )}
```

Comment:
Do we force the parantheses `(` and `)` around jsx components? I'd suggest not to do that.

Reply:
I think we do not force them.

Reply:
There are quite some in the changes above though.